packages/scripts/package.json

```diff
@@ -30,8 +30,7 @@
 	"devDependencies": {
 		"babel-jest": "^22.0.4",
 		"cross-spawn": "5.1.0",
-		"jest": "^22.0.4",
-		"regenerator-runtime": "^0.11.1"
```

Comment:
Unrelated change, this shouldn't be in this PR, it should be updated by removing it in #62